client/my-sites/site-settings/section-seo.jsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SeoForm from 'my-sites/site-settings/form-seo';
+
+export default React.createClass( {
+	displayName: 'SiteSettingsSeo',
+
+	render() {
+		return (
+			<SeoForm site={ this.props.site } />
+		);
+	}
+} );
```

Comment:
This would be a good spot to add page view tracking, though honestly we could probably do without this whole page unless you have further plans for it. It doesn't actually do anything here.